doc/UserGuide.md

```diff
@@ -35,7 +35,8 @@ Format: `help`
  
 ### Adding a person: `add`
 Adds a person to the address book<br>
-Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...` 
+Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`<br>
+Address Format: `a/BLOCK, STREET, UNIT, POSTAL_CODE`
```

Comment:
Good that you updated the user guide :)

However, why not compact the add instructions into a single line instead? e.g. `Format: add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/BLOCK, STREET, UNIT, POSTAL_CODE [t/TAG]...` This can make it less confusing for users who are reading the documentation :)